modules/upgrade-scanner.adoc

```diff
@@ -8,10 +8,26 @@
 You can update Scanner to the latest version by using the `roxctl` CLI.
 
 .Prerequisites
+. If you deploy images from a private image registry, you must first push the new image into your private registry, then edit the following commands to use the name of your private image registry.
+. If you have created custom scanner configurations, you must apply those changes before updating the scanner configuration file:
```

Comment:
I suspect this might be domain knowledge here, but these prerequisites feel like actual steps to me. Is it possible that these steps should be completed earlier in the upgrade process?

Reply:
Alternately, you could move the platform-specific prereqs into the *Procedure* section.